go.sum

```diff
@@ -554,7 +554,7 @@ github.com/tendermint/go-amino v0.16.0 h1:GyhmgQKvqF82e2oZeuMSp9JTN0N09emoSZlb2l
 github.com/tendermint/go-amino v0.16.0/go.mod h1:TQU0M1i/ImAo+tYpZi73AU3V/dKeCoMC9Sphe2ZwGME=
 github.com/tendermint/tendermint v0.34.0-rc4/go.mod h1:yotsojf2C1QBOw4dZrTcxbyxmPUrT4hNuOQWX9XUwB4=
 github.com/tendermint/tendermint v0.34.0-rc6/go.mod h1:ugzyZO5foutZImv0Iyx/gOFCX6mjJTgbLHTwi17VDVg=
-github.com/tendermint/tendermint v0.34.0 h1:eXCfMgoqVSzrjzOj6clI9GAejcHH0LvOlRjpCmMJksU=
+github.com/tendermint/tendermint v0.34.0 h1:CeCs4Q9+sDhdXVLwyKexUq3HhKRBl+uaLZuRSDoL464=
```

Comment:
How did you introduce this change? It is causing a build error.

Reply:
This was introduced cause somehow the project did not build to me due to checksum mismatch. I reverted it anyway